vnext/Microsoft.ReactNative/Modules/Animated/NativeAnimatedNodeManager.h

```diff
@@ -37,7 +37,7 @@ class TransformAnimatedNode;
 class TrackingAnimatedNode;
 class AnimationDriver;
 class EventAnimationDriver;
-class NativeAnimatedNodeManager {
+class NativeAnimatedNodeManager : public std::enable_shared_from_this<NativeAnimatedNodeManager> {
```

Comment:
Trying to remember the history of this.

I remember seeing a fix for cases where this had an intrusive reference count, but something kept a `unique_ptr` to it. Was that fix from the FB repo that already had this change?

Reply:
I think you're thinking of #9158. In that case, we introduced `enable_shared_from_this` on AnimationDriver back in #8418 but never switched the map of `m_activeAnimations` to hold `std::shared_ptrs` instead of `std::unique_ptrs`.

Reply:
But now that you mention it, we probably don't need to add `enable_shared_from_this` because we can just as easily pass the `shared_ptr` to the NativeAnimatedNodeManager from here:
https://github.com/microsoft/react-native-windows/pull/9190/files#diff-0b33c90c397cd63c83ffa33ef49b79adc59e63cc02267923b8f0aa4944445b7eR64

 since we already pass the NativeAnimatedModule's shared_ptr to the same instance of NativeAnimatedNodeManager in a bunch of other places, e.g.: 
 https://github.com/microsoft/react-native-windows/blob/495edc099b2652fadc10db16abe22e1bfd9ca347/vnext/Microsoft.ReactNative/Modules/Animated/NativeAnimatedNodeManager.h#L53-L70